tests/test_attributes.py

```diff
@@ -42,11 +39,12 @@ async def mock_httpx_get(self, **kwargs):
 
     monkeypatch.setattr(httpx.AsyncClient, "get", mock_httpx_get)
 
-    with pytest.warns(UserWarning):
-        response = test_app.get("/attributes/nb:Assessment")
+    response = test_app.get("/attributes/nb:Assessment")
 
     assert response.status_code == status.HTTP_207_MULTI_STATUS
 
+    assert len(caplog.records) > 0
```

Comment:
Worth checking that the logs here include a warning? Maybe:

```suggestion
    assert len(caplog.records) > 0
    any(record.levelname == "WARNING" for record in caplog.records)
```

Reply:
Good idea!